nvme-print-binary.c

```diff
@@ -322,9 +322,10 @@ static void binary_dispersed_ns_psub_log(struct nvme_dispersed_ns_participating_
 	d_raw((unsigned char *)log, sizeof(*log));
 }
 
-static void binary_reachability_groups_log(struct nvme_reachability_groups_log *log)
+static void binary_reachability_groups_log(struct nvme_reachability_groups_log *log, __u64 len)
 {
-	d_raw((unsigned char *)log, sizeof(*log));
+	print_debug("len: %"PRIu64"\n", (uint64_t)len);
```

Comment:
Same comment as with the json output. I don't know if we want to add to every output format debug information. I am fine with having the normal output format annotated for debugging purposes, but adding trippling the debug output seems add unnecessary maintenance burden.

Reply:
Yes agreed then delete the output.